Dial Pad UI bug fixes

DantrazTrev opened the pull request:
Fixes vector-im/element-web/issues/16761 and vector-im/element-web/issues/16737

A scrollbar was implemented in the dialled section to fix the bug

![image](https://user-images.githubusercontent.com/43457420/112279412-e7211a00-8ca9-11eb-8e65-39b9c5072e06.png)


For vector-im/element-web/issues/16761 a basic color change was done as stated in the second option in the issue

![image](https://user-images.githubusercontent.com/43457420/112279708-38c9a480-8caa-11eb-8720-86506bf5791e.png)



Signed-off-by : Ayush Pratap Singh <ayushpratap16@gmail.com>

SimonBrandner commented on res/css/views/voip/_DialPad.scss:
```diff
@@ -30,6 +30,7 @@ limitations under the License.
     text-align: center;
     vertical-align: middle;
     line-height: 40px;
+    color: #15191e;
```
There might be a suitable colour for this in one of the theme files (e.g. `_dark.scss`). Could you please take a look?

DantrazTrev replied:
There are a few i could find but they're not properly labelled .
 
Also they light theme doesn't have these issues so it might get a bit messy , I think I could just update the `$theme-button-bg-color`  in _dark.css . 

So it becomes consistent with light theme. Something like this:
![image](https://user-images.githubusercontent.com/43457420/112285112-f73bf800-8caf-11eb-8a5a-b8bf168cff04.png)


So it becomes consistent(Like orignal Light theme)
![image](https://user-images.githubusercontent.com/43457420/112285030-e12e3780-8caf-11eb-86e5-33b3bf1490e6.png)

SimonBrandner replied:
Changing the value of `$theme-button-bg-color` isn't a good idea. The consistency would be great. If there really isn't anything suitable, you could perhaps add a new colour (both to `_light.scss` and `_dark.scss`)?

DantrazTrev replied:
I just added `$dialpad-button-bg-color` to all the themes(it wasn't compiling without all of them) for the case. I hope this would be alright

turt2live reviewed (approved):
code looks fine, though a designer will have to take a look to confirm the sizing and such.

gaelledel reviewed (changes requested):
Awesome! Could we follow these visual specs here for both Light and dark themes?
https://www.figma.com/file/xqclx5H9SHRh3T4K6S4rGo/Community?node-id=0%3A1

The scroll bar implementation should be removed. Instead I suggest a different interaction mechanism similar to Skype
<img width="647" alt="Screenshot 2021-04-23 at 14 43 21" src="https://user-images.githubusercontent.com/76945183/115880100-4194ce00-a442-11eb-89de-31420b58cd70.png">
Essentially, the first number inside the row would be rolling towards the left everytime a new character is entered. 
Users can then interact with the numbers row using the arrow keys, delete, space bar etc...
You can test out how it works on Skype to have a clearer picture of what I am referring to.

DantrazTrev commented:
Sure I'll just do that. Although just to be clear the extra buttons don't need to be implemented?

DantrazTrev commented:
I think the user interactions might be used in a dial pad used for calling (Which is somewhere in Matrix but not directly accessible in Element ). But not for a dialer in a call that is used for PTSN stuff mostly. ( That's the way Skype did it too) . 


I've added the color schemes and the RTL rolling digits and dropped the scrollbar
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/43457420/116821181-761d2e00-ab96-11eb-8cca-388f4f0e6a01.gif)

dbkr commented:
Fixes https://github.com/vector-im/element-web/issues/17736

dbkr commented:
This now has the necessary approvals, so merging. It predates the github action check, so I'm going to assume it's fine without this.

dbkr commented:
This now has the necessary approvals, so merging. It predates the github action check, so I'm going to assume it's fine without this.
